config/artifacts.json

```diff
@@ -2148,14 +2148,14 @@
 	},
 	"titansThunder":
 	{
-		"bonuses" : [
-			{
+		"bonuses" : {
+			"titanBolt" : {
```

Comment:
Are these bonus names predefined somewhere?

Reply:
No. Just arbitrary name, that can be used as key for overriding them in mods for example. It can be changed safely, the only thing that might break would be mods that change this particular bonus

Creature abilities and multiple other places also use this form, artifacts might be the only place that does not.